Decred Wallet/Features/Wallets/AccountDetailsViewController.swift

```diff
@@ -101,35 +101,35 @@ class AccountDetailsViewController: UIViewController {
     }
     
     func displayAccountBalances() {
-        self.totalBalanceLabel.attributedText = Utils.amountAsAttributedString(amount: self.account.balance?.dcrTotal,
```

Comment:
is this function(func amountAsAttributedString ()) still in use in the project? if not you can remove it

Reply:
I have remove the amountAsAttributedString () because it's in use any more. Please review it again